sdk/lib/atl/atlwin.h

```diff
@@ -320,9 +320,37 @@ class CWindow
         int wndCenterHeight = wndCenterRect.bottom - wndCenterRect.top;
         int wndWidth = wndRect.right - wndRect.left;
         int wndHeight = wndRect.bottom - wndRect.top;
+        int xPos = wndCenterRect.left + ((wndCenterWidth - wndWidth + 1) >> 1);
+        int yPos = wndCenterRect.top + ((wndCenterHeight - wndHeight + 1) >> 1);
+
+        if ((::GetWindowLong(hWndCenter, GWL_STYLE) & WS_POPUP))
```

Comment:
Are you sure `WS_POPUP` is the right check here?
You might want to check for `WS_CHILD` instead.

Reply:
@learn-more  Thanks for your help. Done with the correct check.